exodus_core/analysis/static_analysis.py

```diff
@@ -309,15 +309,23 @@ def _get_icon_from_gplay(self, handle, path):
         from bs4 import BeautifulSoup
         import urllib.request
 
+
         address = 'https://play.google.com/store/apps/details?id=%s' % handle
         text = urllib.request.urlopen(address).read()
         soup = BeautifulSoup(text, 'html.parser')
-        i = soup.find_all('img', {'class': 'cover-image', 'alt': 'Cover art'})
+        i = soup.find_all('img', {'class': 'T75of ujDFqe'})
```

Comment:
I think we should keep the "alt" attribute to make sure we are getting the right picture.
It actually works if you remove the class and only keep the "alt".

Reply:
Good idea. It seems more readable and easier to maintain. Note I did not check what you want to do next.